packages/@azure/servicebus/data-plane/lib/namespace.ts

```diff
@@ -61,17 +50,22 @@ export class Namespace {
   private _context: ConnectionContext;
 
   /**
-   * Instantiates a client pointing to the ServiceBus Queue given by this configuration.
+   * Instantiates a client pointing to the Service Bus Namespace.
    *
    * @constructor
    * @param {ConnectionConfig} config - The connection configuration to create the Namespace.
    * @param {TokenProvider} [tokenProvider] - The token provider that provides the token for
    * authentication. Default value: `SasTokenProvider`.
+   * @param {NamespaceOptions} - Options to create the Namespace.
    */
-  private constructor(config: ConnectionConfig, options?: NamespaceOptions) {
+  private constructor(
+    config: ConnectionConfig,
+    tokenProvider: TokenProvider,
```

Comment:
Comment above says `tokenProvider` has a default but I don't see it applied and it's a required parameter. This is private so it matters less but could consider applying the default at this layer.

Reply:
The note on "default" has been removed